src/lakefs_spec/spec.py

```diff
@@ -276,9 +291,9 @@ def get_file(
             The remote path to download to local storage. Must be a fully qualified lakeFS URI, and point to a single file.
         lpath: str | os.PathLike[str]
             The local path on disk to save the downloaded file to.
-        callback: fsspec.callbacks.Callback
+        callback: Callback
```

Comment:
Hmm, I prefer fully qualified types. Otherwise, someone reading the docstring might think that we define our own callback type somewhere.

Reply:
I agree, but unfortunately, that cannot be configured in `pydoclint`.

One option would be to not use the imported `Callback` type in the actual signature, but rather its fully-qualified name (which `pydoclint` would then enforce in the docstring as well).

Reply:
Did that in ebf7f2f71060e922c45fcc1ef2517a58466de53f